docker-run.sh

```diff
@@ -64,8 +59,6 @@ if [[ "${URL_OAS3_GENERATOR}" ]]; then
   fi
 fi
 
-exec nginx -g 'daemon off;'
```

Comment:
TECHNICAL NOTE EXPLAINING THE CHANGE

>  If you add a custom CMD in the Dockerfile, be sure to include -g daemon off; in the CMD in order for nginx to stay in the foreground, so that Docker can track the process properly (otherwise your container will stop immediately after starting)!

Reply:
Yep, this comment was for reference purposes only. No action on your side was needed.